kiosk/src/Models/Kiosk.ts

```diff
@@ -125,15 +128,15 @@ export class Kiosk {
         this.navigate(KioskState.PlayingGame);
 
         this.siteElements = [];
-        let gamespace = document.getElementsByTagName("BODY")[0];
+        const gamespace = document.getElementsByTagName("BODY")[0];
         while (gamespace.firstChild) {
             this.siteElements.push(gamespace.firstChild);
             gamespace.firstChild.remove();
         }
 
-        let editorEmbedURL = `https://arcade.makecode.com/---run?id=${gameId}&&fullscreen=1`;
+        const editorEmbedURL = `https://arcade.makecode.com/---run?id=${gameId}&&fullscreen=1`;
```

Comment:
```suggestion
        const editorEmbedURL = `https://arcade.makecode.com/---run?id=${gameId}&fullscreen=1`;
```

might want `&noFooter=1` (gets rid of little logo bar that flashes at the bottom sometimes) and `&single=1` (prevents a second sim from popping up when the user uses e.g. radio or multiplayer blocks) as well but that can be added in the future~